documentation/asciidoc/topics/proc_configuring_anti_affinity.adoc

```diff
@@ -0,0 +1,16 @@
+[id='configuring_anti_affinity-{context}']
+= Configuring Anti-Affinity
+Specify where {k8s} schedules pods for your {brandname} clusters to ensure availability.
+
+You can use the , preferring one over the other.
+Alternatively you can configure {ispn_operator} to use only multiple zones when scheduling pods.
+
+.Procedure
+
+. Add the `spec.affinity` field to your **Infinispan** CR.
+. To schedule pods across multiple zones, specify `topology.kubernetes.io/zone` as the value for the `topologyKey` field.
+.. If you use the hostname and zone anti-affinity strategies together, set the `weight` field to give preference to one strategy over another.
+. Change `preferredDuringSchedulingIgnoredDuringExecution` to `requiredDuringSchedulingIgnoredDuringExecution` if you want to schedule {brandname} replica pods:
```

Comment:
How about:

"Change `preferredDuringSchedulingIgnoredDuringExecution` to `requiredDuringSchedulingIgnoredDuringExecution` if you always require the anti-affinity policy to be true. This may prevent all `spec.replicas` from being scheduled if the required number of hosts or zones are not available."

Reply:
yeah, that's better.